frontend/sw360-portlet/src/main/resources/META-INF/resources/html/projects/licenseInfo.jsp

```diff
@@ -72,8 +72,28 @@
           <span aria-hidden="true">&times;</span>
         </button>
       </div>
-      <div class="modal-body">
-						<c:if test="${not empty externalIds}">
+      <div class="modal-body"
+                    style="position: relative; overflow-y: auto; max-height: 400px;">
+                    <c:if test="${not empty projectReleaseRelations}">
+                        <div class="form-group form-check">
+                            <label for="projectRelation"
+                                class="font-weight-bold h3">Uncheck project
+                                release relationships to be excluded:</label>
+                            <c:forEach var="projectReleaseRelation"
+                                items="${projectReleaseRelations}">
+                                <div class="checkbox form-check">
+                                    <label>
+                                        <input id="<%=PortalConstants.SELECTED_PROJECT_RELEASE_RELATIONS%>" name="releaseRelationSelection" type="checkbox"
```

Comment:
Please check if `ID `attribute is required or can be made unique , since it is inside `for `loop is creates multiple input fields with same `ID`.

Reply:
removed the ID